packages/components/src/components/linear-progress/linear-progress-container.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { LinearProgress } from './linear-progress.jsx';
+import { LinearProgress } from './linear-progress';
+
+type TLinearProgressContainer = {
+    timeout: number;
+    action: () => void;
+    render: (prop: number) => number;
+    className?: string;
+    should_store_in_session?: boolean;
+    session_id: string;
+};
 
 const LinearProgressContainer = React.forwardRef(
-    ({ timeout, action, render, className, should_store_in_session, session_id }, ref) => {
-        const current_progress_timeout = sessionStorage.getItem(`linear_progress_timeout_${session_id}`);
+    ({ timeout, action, render, className, should_store_in_session, session_id }: TLinearProgressContainer, ref) => {
+        const current_progress_timeout = Number(sessionStorage.getItem(`linear_progress_timeout_${session_id}`));
 
-        const popup_timeout = !current_progress_timeout ? timeout / 1000 : current_progress_timeout;
+        const popup_timeout = !current_progress_timeout ? timeout / 1000 : Number(current_progress_timeout);
```

Comment:
@jim-deriv please consider not converting current_progress_timeout to number the second time after it's been already converted to it at line 15  🙂